### DISCUSSION TITLE
no logs showen in kibana and so

### DISCUSSION THREAD

**Discussion by C-ajal** [Security-Onion-Solutions/securityonion#10966] [General]
i send the syslog from the firewall to the manager server and is not showen in any log  ican see the logs by use tcpdump but not showing in kibana and so 
i put the subnet in so-allow list  but there is no logs dashboard

**Comment by InfosecGoon** [comment]
You're sending logs via syslog on 514/UDP, and you've added the sending IP to your configuration with so-allow, but they're not visible?

Do you see anything in Hunt if you search for the sending address on the firewall? Something like:

`"X.X.X.X" | groupby event.module event.dataset`

**Reply by C-ajal** [reply]
There is nothing I have tried and I searched and did not find a solution  for this problem
